pyScienceMode2/sciencemode.py

```diff
@@ -75,30 +75,36 @@ def __init__(self, port: str, show_log: bool = False, with_motomed: bool = False
         self.packet_count = 0
         self.reha_connected = False
         self.show_log = show_log
-
         self.time_last_cmd = 0
         self.packet_send_history = []
-
         self.read_port_time = 0.0
         self.last_ack = None
         self.last_init_ack = None
         self.motomed_values = None
-        self.last_phase_result = None
-        self._motomed_command_done = True
-        self.is_motomed_connected = with_motomed
         self.max_motomed_values = 100
         self.max_phase_result = 1
         self.__thread_watchdog = None
         self.lock = threading.Lock()
         self.motomed_done = threading.Event()
         self.is_phase_result = threading.Event()
-        self.command_sent = threading.Event()
         self.event_ack = threading.Event()
-        self.__motomed_thread_started = False
+        self.last_phase_result = None
+        self._motomed_command_done = True
+        self.is_motomed_connected = with_motomed
+        self.__comparison_thread_started = False
         self.__watchdog_thread_started = False
+        self.info_send = []  # Command sent to the rehastim
```

Comment:
if it is a command then call it command not info.

Reply:
Renamed